setup/freebsd/ansible-vars.yaml

```diff
@@ -4,8 +4,9 @@ init_script_path: /usr/local/etc/rc.d/jenkins
 ssh_users:
   - rvagg
   - jbergstroem
+  - joaocgreis
```

Comment:
Copied from our linter config (which is also freebsd). I reckon this list should be expanded; probably with @orangemocha.